docs/tutorials/index.rst

```diff
@@ -20,11 +35,26 @@ port your existing code to the Determined environment:
 |                                 | port a ``tf.keras`` model to Determined.                     |
 +---------------------------------+--------------------------------------------------------------+
 
+Go Further
+==========
+
+Visit the `Training API Guides
+<https://docs.determined.ai/latest/training/apis-howto/overview.html>`_ for in-depth guides that
```

Comment:
you are using absolute urls everywhere, can we use relative urls instead?  That way when people are seeing the docs on their own local cluster (which might not have general internet access) the links don't direct them to totally different urls?  Also, it keeps somebody from following links that accidentally bring them to a newer version of the docs than what their cluster is running.

Reply:
all are now cross refs